examples/azure/packages/function-app-powershell/hello-world/run.ps1

```diff
@@ -0,0 +1,8 @@
+param($req, $TriggerMetadata)
+
+$name = $req.Query.Name
+
+Push-OutputBinding -Name Response -Value ([HttpResponseContext]@{
+  StatusCode = [System.Net.HttpStatusCode]::OK
+  Body = "Hello $name!"
```

Comment:
QoL: really minor, would like having

```suggestion
  Body = "Hello $name from Powershell!"
```

similar to what we have [here](https://github.com/THEY-Consulting/they-terraform/blob/main/examples/azure/packages/function-app-typescript/hello-world/index.ts#L4)